lib/sioFb.js

```diff
@@ -49,13 +51,18 @@ class SioFb {
   };
 
   getNewStrips() {
-    return this.getLatestStrips()
+    let lastPolledDate;
+    console.log('Polling for new strips...');
+    return this.repository.getLastPolledDate()
+      .selectMany(date => {
+        console.log('Polling last run on %s.', date);
+        lastPolledDate = date;
+        return this.getLatestStrips();
```

Comment:
if you move the filter here you don't  need to scope it outside and it reads a bit nicer:

eg:

`.selectMany(date => getStripsAfter(date))`

getStripAfter =>

`return this.getLatestStrips().filter()`